app/components/filters/date-range-filter.ts

```diff
@@ -200,9 +202,36 @@ export default class DateRangeFilterComponent extends Component<Signature> {
     }
   }
 
+  errorMessages = A<string>([]);
+
   updateQueryParamsIfValid() {
+    // if the start is before 2015 or the end after today + 100 years
     if (this.hasBothDates || this.hasNoDates) {
+      if (this.start && new Date(this.start) < new Date('2015-01-01')) {
```

Comment:
@snenenenenenene **Blocker:** start and end dates are different in class and template

Reply:
@aliokan I don't completely understand the problem, could you elaborate?

Reply:
@snenenenenenene In `app/components/filters/date-range-filter.ts` the date range is defined from `2015-01-01` to `2100-01-01`, but in `app/components/filters/date-range-filter/date-input.hbs` is from `2000-01-01` to `2099-12-31`

Reply:
@aliokan should be fixed!